README.md

```diff
@@ -42,6 +42,31 @@
 $ composer require meilisearch/meilisearch-laravel-scout
 ```
 
+### HTTP Client
+
+You could use any [PSR-18](https://www.php-fig.org/psr/psr-18/) compatible client to use with this SDK. No additional configurations required.<br>
+A list of compatible HTTP clients and client adapters can be found at [php-http.org](http://docs.php-http.org/en/latest/clients.html).
+
+If you use **Laravel 8** you can skip this section as laravel pre-install Guzzle 7 by default.
+
```

Comment:
Could we add a line for Guzzle 7:

```bash
$ composer require guzzlehttp/guzzle
```

For users using Laravel < 8 but who want to use guzzle 7

Reply:
Is it ok to you ?